celestia/peers.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
Can you simply not change this file at all, please? Otherwise there are merge conflicts etc.